### PR TITLE
Feature/5 api implement db

### PR DESCRIPTION
**Pull request by chainowsky**

### CODE REVIEW

File: API/Data/ApiDbContext.cs [hunk 1]
```diff
@@ -9,9 +9,17 @@ public class ApiDbContext : IdentityDbContext<AppUser>
         public ApiDbContext(DbContextOptions<ApiDbContext> options) : base(options)
         {
         }
+        public DbSet<Match> Matches => Set<Match>();
+        public DbSet<Player> Players => Set<Player>();
+        public DbSet<Position> Positions => Set<Position>();
+        public DbSet<Team> Teams => Set<Team>();
+        public DbSet<Tournament> Tournaments => Set<Tournament>();
+
+
```
Comment (thestahan) [inline API/Data/ApiDbContext.cs:18]: Let's leave only one blank line here

File: API/Domain/Position.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace API.Domain;
+
+public class Position
+{
+    public int Id { get; set; }
+    public string Name { get; set; } = string.Empty;
+
```
Comment (thestahan) [inline API/Domain/Position.cs:7]: Let's stick to the version with no blank lines between "normal" properties and ICollection

File: API/Domain/Team.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿namespace API.Domain;
+
+public class Team
+{
+    public int Id { get; set; }
+    public string Name { get; set; } = string.Empty;
+    public string? City { get; set; }
+    public string? Coach { get; set; }
+
```
Comment (thestahan) [inline API/Domain/Team.cs:9]: As above

File: API/Domain/Team.cs [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+public class Team
+{
+    public int Id { get; set; }
+    public string Name { get; set; } = string.Empty;
+    public string? City { get; set; }
+    public string? Coach { get; set; }
+
+    public ICollection<Tournament> Tournaments { get; set; }
+    public ICollection<Player> Players { get; set; }
+
```
Comment (thestahan) [inline API/Domain/Team.cs:12]: Here's also an unnecessary line

File: API/Data/ApiDbContext.cs [hunk 5]
```diff
@@ -9,9 +9,17 @@ public class ApiDbContext : IdentityDbContext<AppUser>
         public ApiDbContext(DbContextOptions<ApiDbContext> options) : base(options)
         {
         }
+        public DbSet<Match> Matches => Set<Match>();
```
Comment (thestahan) [inline API/Data/ApiDbContext.cs:12]: For better clarity, we should always leave one blank line between methods and other things; tldr: add one blank line above

**Review by thestahan — CHANGES_REQUESTED** [review]
Looks great, only some minor formatting issues need to be addressed.

**Review by chainowsky — COMMENTED** [review]
improved

**Review by thestahan — APPROVED** [review]

**Pull request merged**